ide-support/revsblibrary.livecodescript

```diff
@@ -1430,23 +1536,26 @@ function revSBAdditionalExternalsList pPlatform
          set the itemdelimiter to "."
          set the linedelimiter to slash
          repeat for each element tExtPath in tRawList
-            put tExtPath into sExternalsLocationA[pPlatform][item 1 of line -1 of tExtPath]
+            local tExtension, tExternalName
+            put item 2 of line -1 of tExtPath into tExtension
+            put item 1 of line -1 of tExtPath into tExternalName
```

Comment:
This was a bug in the existing implementation, but do you think this should be
```
put item -1 of line -1 of tExtPath into tExtension
put item 1 to -2 of line -1 of tExtPath into tExternalName
```
in case the external name contains a `.`?